crypto/ec/ec_mult.c

```diff
@@ -236,6 +238,21 @@ int ec_scalar_mul_ladder(const EC_GROUP *group, EC_POINT *r,
         }
     }
 
+    /*
+     * Create random values for BN_consttime_swap_randomized, and for
+	 * constant-time swapping of Z_is_one in EC_POINT_SWAP below
+	 * This randomizatin is needed to prevent side channel attacks
```

Comment:
Something odd happening with the indentation here

Reply:
Typo: randomizatin

Reply:
fixed typo and indentation (tab/space) in the new PR.